lib/Activity.py

```diff
@@ -55,7 +55,9 @@ class ActivityError(Exception):
     'state': None,
     'sessionid': None,
     'status': None,
-    'comment': None
+    'comment': None,
+    'media_dir': None,
+    'command': None,
```

Comment:
I think "command" is good, but I think it might _also_ be good to nab a copy of the cli args.